runtime/tests/doughnut.rs

```diff
@@ -152,10 +181,104 @@ fn it_fails_when_module_name_is_empty() {
 
 #[test]
 fn it_fails_when_module_name_and_prefix_are_empty() {
-	let cennznut = make_cennznut("attestation", "attest");
+	let cennznut = make_runtime_cennznut("attestation", "attest");
 	let doughnut = make_doughnut("cennznet", cennznut.encode());
 	assert_err!(
 		verify_dispatch(&doughnut, "-", "attest"),
 		"error during module name segmentation"
 	);
 }
+
+#[test]
+fn it_fails_when_using_contract_cennznut_for_runtime() {
+	let cennznut = make_contract_cennznut([0x11; 32].into());
```

Comment:
[Suggestion] Make global var for`[0x11; 32]`, same with `const TEST_HOLDER: ...`, like `CONTRACT_ADDRESS`